### PR TITLE
Sean flight filters

### PR DESCRIPTION
**Pull request by sean-darsie**

### CODE REVIEW

File: src/app/traveler/flights/flights.component.html [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+        [(ngModel)]="customPrice"
+      />
+    </fieldset>
+  </div>
+</div>
+<table class="table table-hover">
+  <thead>
+    <tr>
+      <th scope="col">Depart From</th>
+      <th scope="col">Arrive To</th>
```
Comment (justin-obrien-smoothstack) [inline src/app/traveler/flights/flights.component.html:106]: "Arrive At" would be better than "Arrive To".

File: src/app/traveler/flights/flights.component.html [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+            <tr class="table-dark">
+              <th scope="row">Departure Airport:</th>
+              <td>{{ selectedFlight.departAirport }}</td>
+            </tr>
+            <tr class="table-dark">
+              <th scope="row">Arrival Airport:</th>
+              <td>{{ selectedFlight.arriveAirport }}</td>
+            </tr>
+            <tr class="table-dark">
+              <th scope="row">Flight Time:</th>
```
Comment (justin-obrien-smoothstack) [inline src/app/traveler/flights/flights.component.html:179]: This is potentially ambiguous. "Departure Time" would be more precise.

File: src/environments/environment.ts [hunk 3]
```diff
@@ -5,17 +5,22 @@
 export const environment = {
   production: false,
   travelerBackendUrl: 'http://localhost:8080/traveler',
+  counterUrl: 'http://localhost:8080/counter',
+  loginUrl: 'http://localhost:8080/login',
```
[the remark this answers is not in the document]
Comment (sean-darsie) [reply src/environments/environment.ts:9]: i must have made that mistake when fixing a merge.

File: src/app/traveler/flights/flights.component.ts [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  pageSize = 10;
+  filterMetadata = { count: 0 };
+
+
+  creditCardForm = new FormGroup({
+    cardNumber: new FormControl('', [Validators.required as any]),
+    expMonth: new FormControl('', [Validators.required as any]),
+    expYear: new FormControl('', [Validators.required as any]),
+    cvc: new FormControl('', [Validators.required as any]),
+  });
```
[the remark this answers is not in the document]
Comment (sean-darsie) [reply src/app/traveler/flights/flights.component.ts:39]: I'll do that in the next pr

File: src/app/common/h/sort-flights-airports.ts [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+export class SortByFlightPrice implements PipeTransform {
+    transform(items: any[], price, filterMetadata): any {
+        if (price === 100) {
+            filterMetadata.count = items.length;
+            return items;
+        }
+
+        if (price) {
+        const filteredArray = items.filter(
+          (item) => item.price < price
```
[the remark this answers is not in the document]
Comment (sean-darsie) [reply src/app/common/h/sort-flights-airports.ts:60]: removed this file in  next pr.

**Comment by trevorhuis-smoothstack** [comment]
I am going to wait for an update before reviewing since Justin caught a ton of stuff.

File: src/index.html [hunk 6]
```diff
@@ -14,7 +14,7 @@
   
   <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
   <script type="text/javascript">
-    Stripe.setPublishableKey('pk_test_51Guj65Fb3TAD5KLT94lDvAoFWPcLphSSna40tyv7hCbT8m14pxaItIRXf4y5N33ZaYEU8cqVjXJ7I8lteoAUrmrE00E3zXAfTw');
+    Stripe.setPublishableKey('pk_test_51GvUChBYMFlMJbBRvrWM7yZJHJhVExdReQ2A5K0uaKTidkmqRcnY48fr6VmnK9csVNOwkiH0xetgz36Gcvql6IF20098oe4tpg');
   </script>
```
[the remark this answers is not in the document]
Comment (sean-darsie) [reply src/index.html]: that is going to be removed in the next pr. This pr focused on the features of the dashboard and bookings.

**Pull request merged**